sound/soc/sof/card-tplg-ops.c

```diff
@@ -0,0 +1,114 @@
+// SPDX-License-Identifier: (GPL-2.0-only OR BSD-3-Clause)
+//
+// This file is provided under a dual BSD/GPLv2 license.  When using or
+// redistributing this file, you may do so under either license.
+//
+// Copyright(c) 2025 Intel Corporation.
+//
+
+#include <linux/device.h>
+#include <linux/errno.h>
+#include "sof-priv.h"
+#include "topology.h"
+
+enum tplg_device_id {
```

Comment:
I guess these are SDCA devices types ? I guess we can incrementally reuse this or comment as we align with SDCA

Reply:
> I guess these are SDCA devices types ? I guess we can incrementally reuse this or comment as we align with SDCA

They are tplg devices not just SDCA devices types. Currently, we will create one mini-topology for one tplg device.